### PR TITLE
Update various plugins which have security updates

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

File: attributes/plugins.rb [hunk 1]
```diff
@@ -39,6 +39,7 @@
   icon-shim:2.0.3
   instant-messaging:1.35
   ircbot:2.27
+  jackson2-api:2.7.3
```
Comment (doublej472) [inline attributes/plugins.rb]: Why is this brand new plugin here?

Comment (leel8on) [inline attributes/plugins.rb]: `github-api` now requires it.

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.8.7.  Have a nice day!

**Pull request merged**